README.md

```diff
@@ -3,16 +3,34 @@
 Sandbox Drupal URL: https://performance.ddev.site
 Sandbox Frontend URL: http://localhost:3000
 
-## Stand up a local sandbox
+## Initial setup for local sandbox
 1. ddev start
 2. dev composer install
-3. ddev drush config:import --source=config
-4. ddev drush uli
+
+### If no local database exists
+1. In config/core.extension.yml, Change the lines that read standard: 1000 and profile: standard to minimal: 1000 and profile: minimal (to switch the configured install profile to minimal)
+2. ddev drush sql:drop && drush si --existing-config
+3. ddev drush cim
```

Comment:
This should not be necessary, since it's what happens when you run  --existing-config

The whole 3-point session can be reduced to  just this:

1. `ddev drush si --existing-config`

Reply:
Plus, if/when we can presume everyone has Airtable access, Key setup instructions and `ddev migrate` should go here. Example: https://github.com/CivicActions/pgov-cms/pull/12